### PR TITLE
fixes stored procedure for looking up account by name

### PR DESCRIPTION
**Pull request by nellshamrell**
Fixes a release blocking bug for the data migration.

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-sessionsrv/src/migrations/accounts.rs [hunk 1]
```diff
@@ -53,7 +53,7 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                     account_name text
                  ) RETURNS SETOF accounts AS $$
                      BEGIN
-                        RETURN QUERY SELECT * FROM accounts WHERE name = name;
+                        RETURN QUERY SELECT * FROM accounts WHERE name = account_name;
```
Comment (reset) [inline components/builder-sessionsrv/src/migrations/accounts.rs]: Holy shit, good catch

**Comment by reset** [comment]
tenor-49671408

@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/233817475).

**Comment by adamhjk** [comment]
tenor-137402751

You're welcome ;)

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/233817475).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**